File: app/models/location.rb
# == Schema Information
#
# Table name: locations
#
#  id       :integer          not null, primary key
#  name     :string           not null
#  country  :string           not null
#  region   :string
#  lat      :decimal(10, 6)   not null
#  lon      :decimal(10, 6)   not null
#  asl      :integer          not null
#  timezone :string
#

class Location < ApplicationRecord
  include ActiveModel::Serializers::JSON
  has_many :sun_data_days
end

File: frontend/app/models/location.js
import Ember from 'ember';
import DS from 'ember-data';

export default DS.Model.extend({
  name: DS.attr(),
  country: DS.attr(),
  region: DS.attr(),
  lat: DS.attr(),
  lon: DS.attr(),
  asl: DS.attr(),
  timezone: DS.attr(),

  displayName: Ember.computed('name', function () {
    return this.get('name');
  })
});
